### PR TITLE
add php image documentation

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@adellape @CowboysFan @bfallonf ptal.

@soltysh @jhadvig ptal also.  I think this should become (after any comments you have) the new template for our other STI image doc pages.  (stripping out all the "how to build the image" info)

**Comment by adellape** [comment]
FYI @tpoitras, since you've worked on most of the image pages so far.

File: using_images/sti_images/php.adoc [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+This image is available on DockerHub. To download it:
+
+****
+`$ docker pull openshift/php-55-centos7`
+****
+
+To use these images, you can either access them directly in those registries or push them into your OpenShift docker registry.  
+In addition, it is recommended that you create an ImageStream that points to the image, either in your docker registry
+or at the external location.  Your OpenShift resources can then reference the ImageStream.  You can find example ImageStream definitions for all the provided OpenShift images https://github.com/openshift/origin/examples/image-streams[here].
```
Comment (soltysh) [inline using_images/sti_images/php.adoc]: Actually php is not there, missing PR?

**Comment by soltysh** [comment]
LGTM, I'll be working hopefully later this week on simplifying the other image docs, per Dan's request (FYI @tpoitras, I'll try to reach you and sync with you before actually starting doing this).

File: using_images/sti_images/php.adoc (hunk 1 above)
Comment (bparees) [inline using_images/sti_images/php.adoc]: yes, good catch, thanks.

Comment (adellape) [inline using_images/sti_images/php.adoc]: This link also I assume should be: https://github.com/openshift/origin/tree/master/examples/image-streams ?

**Comment by adellape** [comment]
@bparees LGTM, just the 1 comment inline.

File: using_images/sti_images/php.adoc (hunk 1 above)
Comment (bparees) [inline using_images/sti_images/php.adoc]: yes..fixed, thanks.

**Comment by bparees** [comment]
@adellape thanks, updated..merge at will.  i've got another PR to fix the imagestream.json issue @soltysh pointed out.

**Pull request merged**